common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -624,7 +624,7 @@ def definition_from_xml(cls, xml_object, system):
         ]
         definition = {
             attr_name: json.loads(attr_value)
-            for attr_name, attr_value in xml_object.attrib
+            for attr_name, attr_value in xml_object.attrib.iteritems()
```

Comment:
hmmm, I think `items()` works. (it's technically a little less performant because it has to build out full lists, but there's few attributes to iterate over anyway...)